src/stories/DatePicker.stories.tsx

```diff
@@ -43,6 +43,8 @@ stories.add("All Props", () => {
       disabled={boolean("disabled", false)}
       autoResponsive={boolean("auto responsive", true)}
       disabledBeforeToday={boolean("disabled before today", false)}
+      disabledBeforeDate={text("disabled before date", disabledBeforeDate)}
+      disabledAfterDate={text("disabled before date", disabledAfterDate)}
```

Comment:
And I think it should be `disabled after date` ✌️

Reply:
whoops!